### DISCUSSION TITLE
Navigating elements with ↑↓ ←→ (when the canvas is focused)

### DISCUSSION THREAD

**Discussion by mmiroshnikov** [webstudio-is/webstudio-community#167] [Ideas]
In your current implementation it works only when the navigator panel is active, but when I click an element on the canvas and want to edit it's parent or siblings it doesn't work.

Please check Webflow or Figma. It makes navigating complex layer structures SO much easier!

I really like WF way: ←→ switches betweeb siblings, ↑↓ moves up/down in the herarchy.

**Comment by kof** [comment]
Great idea!
